frontend-react/src/content/resources/api-programmers-guide/ReportStreamApi.mdx

```diff
@@ -0,0 +1,66 @@
+import { CardGroup, Card, CardBody, CardHeader, ProcessList, ProcessListItem, ProcessListHeading  } from "@trussworks/react-uswds";
+
+# ReportStream API 
+
+<p className="usa-intro text-base">
+    By using the ReportStream Restful (REST) API, your data will automatically be filtered, transformed, batched, and forwarded to public health departments.
+</p>
+
+<CardGroup>
+    <Card gridLayout={{col: "fill"}}>
+        <CardHeader><span className="font-sans-lg">[Getting started](getting-started)</span></CardHeader>
+        <CardBody>Resources and support to set up your connection and format your data. Our onboarding guide will walk you through the steps needed to get started.</CardBody>
+    </Card>
+    <Card gridLayout={{col: "fill"}}>
+        <CardHeader><span className="font-sans-lg">[Documentation](documentation)</span></CardHeader>
+        <CardBody>Details on our data model and sample responses, inputs and outputs to assist you in our set up and use of the ReportStream API.</CardBody>
+    </Card>
+</CardGroup>
+
+<hr/>
```

Comment:
Why not use --- here instead of <hr /> ?

Reply:
Be sure to use backticks around the code you're quoting so that it doesn't get interpreted literally. This was before I remembered the MD syntax. Thanks for pointing this out.